css/style/navigation.css

```diff
@@ -0,0 +1,38 @@
+.nav{
+    width:inherit;
+    height: auto;
+    padding:.2rem 0rem;
+}
+.nav-lg{
+    display:grid;
+    justify-content: space-between;
+    grid-template-columns: repeat(2,1fr);
+}
+.dropdown{
+    position: relative;
+    margin:0rem 1rem;
+}
+.dropbtn{
+    background-color: var();
```

Comment:
hey, Why are you using css variable if there is no value in it?

Reply:
That's by mistake, corrected it